src/pages/workspaces/workspace/notebooks/NotebookLauncher.js

```diff
@@ -452,6 +452,23 @@ const JupyterFrameManager = ({ onClose, frameRef }) => {
   return null
 }
 
+const PeriodicCookieSetter = ({ namespace, clusterName }) => {
+  const signal = useCancellation()
+
+  const periodicallySetCookie = async () => {
+    while (!signal.aborted) {
+      withErrorIgnoring(() => Ajax().Jupyter.notebooks(namespace, clusterName).setCookie())
+      await Utils.delay(900000)
```

Comment:
Nit: Factor for clarity, i.e. `15 * 60 * 1000`.
Also, the PR description says 5 minutes but 900000 is 15 minutes. Which should it be?

Reply:
Ah good catch - I updated the code but forgot to update the PR description. We've decided 15 minutes would be better since